gtda/diagrams/_metrics.py

```diff
@@ -51,204 +65,273 @@ def heats(diagrams, sampling, step_size, sigma):
     if np.array_equal(diagrams_1, diagrams_2):
-        unnorm_dist = squareform(pdist(betti_curves_1, 'minkowski', p=p))
+        unnorm_dist = squareform(pdist(betti_curves_1,
+                                       "minkowski", p=p))
         return (step_size ** (1 / p)) * unnorm_dist
     betti_curves_2 = betti_curves(diagrams_2, sampling)
-    unnorm_dist = cdist(betti_curves_1, betti_curves_2, 'minkowski', p=p)
+    unnorm_dist = cdist(betti_curves_1, betti_curves_2,
+                        "minkowski", p=p)
     return (step_size ** (1 / p)) * unnorm_dist
 
 
-def landscape_distances(diagrams_1, diagrams_2, sampling, step_size,
-                        p=2., n_layers=1, **kwargs):
+def landscape_distances(
+    diagrams_1, diagrams_2, sampling, step_size, p=2.0, n_layers=1, **kwargs
+):
     n_samples_1, n_points_1 = diagrams_1.shape[:2]
     n_layers_1 = min(n_layers, n_points_1)
     if np.array_equal(diagrams_1, diagrams_2):
-        ls_1 = landscapes(diagrams_1, sampling, n_layers_1).reshape(
-            n_samples_1, -1)
-        unnorm_dist = squareform(pdist(ls_1, 'minkowski', p=p))
+        ls_1 = landscapes(diagrams_1, sampling,
+                          n_layers_1).reshape(n_samples_1, -1)
+        unnorm_dist = squareform(pdist(ls_1, "minkowski", p=p))
         return (step_size ** (1 / p)) * unnorm_dist
     n_samples_2, n_points_2 = diagrams_2.shape[:2]
     n_layers_2 = min(n_layers, n_points_2)
     n_layers = max(n_layers_1, n_layers_2)
-    ls_1 = landscapes(diagrams_1, sampling, n_layers).reshape(
-        n_samples_1, -1)
-    ls_2 = landscapes(diagrams_2, sampling, n_layers).reshape(
-        n_samples_2, -1)
-    unnorm_dist = cdist(ls_1, ls_2, 'minkowski', p=p)
+    ls_1 = landscapes(diagrams_1, sampling,
+                      n_layers).reshape(n_samples_1, -1)
+    ls_2 = landscapes(diagrams_2, sampling,
+                      n_layers).reshape(n_samples_2, -1)
+    unnorm_dist = cdist(ls_1, ls_2, "minkowski", p=p)
     return (step_size ** (1 / p)) * unnorm_dist
 
 
 def bottleneck_distances(diagrams_1, diagrams_2, delta=0.01, **kwargs):
-    return np.array([[
-        bottleneck_distance(
-            diagram_1[diagram_1[:, 0] != diagram_1[:, 1]],
-            diagram_2[diagram_2[:, 0] != diagram_2[:, 1]], delta)
-        for diagram_2 in diagrams_2] for diagram_1 in diagrams_1])
-
-
-def wasserstein_distances(diagrams_1, diagrams_2, p=2, delta=0.01,
-                          **kwargs):
-    return np.array([[
-        wasserstein_distance(
-            diagram_1[diagram_1[:, 0] != diagram_1[:, 1]],
-            diagram_2[diagram_2[:, 0] != diagram_2[:, 1]], p, delta)
-        for diagram_2 in diagrams_2] for diagram_1 in diagrams_1])
-
-
-def heat_distances(diagrams_1, diagrams_2, sampling, step_size,
-                   sigma=1., p=2., **kwargs):
-    heat_1 = heats(diagrams_1, sampling, step_size, sigma).\
-        reshape(diagrams_1.shape[0], -1)
+    return np.array(
```

Comment:
This linting adds really too many lines. 79 characters is already quite low on modern screens if you need to scroll 10 min to look up a simple code, this is not too efficient.

Reply:
Changed